gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1322,24 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+uintptr_t
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env,UDATA walkFlags, uintptr_t walkReason, MM_HeapWalkerObjectFunc walkFunction)
+{
+	HeapSizes counter;
+	counter.freeBytes = 0;
+	counter.objectBytes = 0;
+	Trc_MM_ClearHeap_Entry(env->getLanguageVMThread());
+	OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+	_heapWalker->allObjectsDo(env, walkFunction, &counter, walkFlags, false, false, true);
+	U_64 startTime = omrtime_hires_clock();
+	_extensions->globalGCStats.fixHeapForWalkTime += omrtime_hires_delta(startTime, omrtime_hires_clock(), OMRPORT_TIME_DELTA_IN_MICROSECONDS);
+	Assert_MM_true(_extensions->globalGCStats.fixHeapForWalkReason == FIXUP_DEBUG_TOOLING);
+	_extensions->globalGCStats.fixHeapForWalkReason = walkReason;
+	Trc_MM_ClearHeap_Exit(env->getLanguageVMThread(), counter.freeBytes, counter.objectBytes);
```

Comment:
For simple methods we don't need entry and exit trace points, but just one of type  TraceEvent.

Often we would not have only method name in the name of the trace point, but also the class name, but since I anticipate in future we might move this method to base MM_Collector class to support clearing for Balanced GC too, this time let's leave it without class name.

Reply:
keep a  simple print here for some time